.github/workflows/test.yaml

```diff
@@ -108,25 +126,23 @@ jobs:
         # jupyterlab.browser_check with jupyterlab 2 and a modern version of
         # python (3.11+).
         #
-        if: ${{ !(matrix.jupyterlab-version == '2' && startsWith(matrix.python-version, '3.1')) }}
+        if: ${{ !(matrix.jupyterlab-version == '2' && startsWith(matrix.python-version, '3.11')) }}
         run: |
+          set -eux
           jupyter labextension list
-          jupyter labextension list 2>&1 | grep -ie '@jupyterhub/jupyter-server-proxy.*OK.*'
+          jupyter labextension list 2>&1 | grep -iE '@jupyterhub/jupyter-server-proxy.*OK.*'
           python -m jupyterlab.browser_check
 
-      - name: Install Acceptance test dependencies
-        run: |
-          # the acceptance test requires notebook to run
-          pip install "notebook<7" "robotframework-jupyterlibrary>=0.4.2"
-
       - name: Run acceptance tests
         run: |
-          pytest -k acceptance -s
+          set -eux
```

Comment:
I don't want to introduce something that isn't needed, and I don't think this should be needed.

1. -e --- We will already error by default in github actions shell scripts
2. -u --- About unset environment variables, I think maybe we won't error if we have an unset environment variable, but we don't provide on in this script section either.
3. -x --- GitHub actions already print stuff we do

If we do want to change something, we can do it in the `defaults.run` section I think.

Reply:
```suggestion
```